packages/scrum-client/src/content/Profile/Profile.js

```diff
@@ -20,9 +33,42 @@ export default function Profile() {
     generateReferralLink(userDetails);
   }, [userDetails]);
 
+  const deleteProfile = async event => {
+    event.preventDefault();
+    try {
+      Request.post(
+        `${API_CONSTANTS.API_BASE_URL}${END_POINTS.API}${END_POINTS.DELETE_PROFILE}`,
+        userDetails
+      ).then(response => {
+        if (response) {
+          logout(dispatch);
+        } else {
+          // TODO what to do?
+          console.log('profile not deleted?');
+        }
+      });
+    } catch (e) {
+      console.log('error', e);
+    }
+  };
+
   return (
     <Container className="Login">
-      <Form>
+      <Modal show={showModal} onHide={modalClose}>
+        <Modal.Header closeButton>
+          <Modal.Title>Sure?</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>This operation is final and unreversible, chose wisely!</Modal.Body>
+        <Modal.Footer>
+          <Button variant="danger" onClick={deleteProfile}>
```

Comment:
for future references , it is a common practice  to name the functions assigned to an event as handlers, 
Example `handleDeleteProfile`